src/app/modules/group/pages/group-details/group-details.component.html

```diff
@@ -14,7 +14,7 @@
     >
     </alg-group-indicator>
     <!-- tabs -- if only the first tab is visible, do not show the tab bar -->
-    <div [hidden]="!group.isCurrentUserManager && group.currentUserMembership === 'none' && !adminTab.isActive && !accessTab?.isActive">
+    <div [hidden]="!group.isCurrentUserManager && group.currentUserMembership === 'none' && !adminTab.isActive && !accessTab?.isActive && !compositionTab.isActive">
```

Comment:
I think the settings tab can be included as well:
https://dev.algorea.org/branch/bugfix/no-tab-when-navigating-to-group/en/#/groups/by-id/6884210558506774905;path=/details/settings

Reply:
Fixed